README.md

```diff
@@ -112,7 +112,7 @@ Do NOT run the node as root.
 The following lines show `[sudo]` as optional. Adjust accordingly. 
 
 ```bash
-[sudo] echo "deb http://ftp.debian.org/debian jessie-backports main" | tee -a /etc/apt/sources.list
+[sudo] echo "deb http://ftp.debian.org/debian jessie-backports main" | [sudo] tee -a /etc/apt/sources.list
```

Comment:
I am not sure this change is correct. The first command does not need sudo, it is the second (tee) that accesses a restricted file.

Reply:
I think you are right. There is no [sudo] on Debian Jessie instructions.